services/ingestor/src/main/kotlin/company/thewebhook/ingestor/Application.kt

```diff
@@ -47,16 +50,28 @@ fun Application.module() = launch {
             .lowercase()
             .split(",")
 
+    val serverHostRegexString =
+        environment.config.propertyOrNull("ingestor.validation.serverHostRegex")?.getString()
+            ?: throw ServerHostRegexException("Serverhost regex string not provided")
```

Comment:
Why declare a separate `ServerHostRegexException`?

Reply:
Not a particular reason, I thought it might be more informative. We can change it to a normal exception too though. Shall I change it?

Reply:
What about `ApplicationConfigException`? That will be generic enough that we can use in other instances where there's a problem with the application config.

Also change the exception message to `ingestor.validation.serverHostRegex cannot be empty`.

Reply:
Okay ye that'd be better